nz_store/tests/test_models/test_product_model.py

```
@@ -0,0 +1,605 @@
+        import time
+
+        time.sleep(0.01)
+
+        self.product.name = "Updated Product Name"
+        self.product.save()
+
+        self.assertGreater(self.product.updated_at, original_updated_at)
+
+    def test_product_cascade_delete_with_category(self):
+        """Test that products are deleted when category is deleted."""
+        self.category.id
```

Ruff (0.11.9)

Found useless expression. Either assign it to a variable or remove it.

(B018)